cpython-windows/build.py

```diff
@@ -233,26 +237,34 @@ def find_vs_path(path, msvc_version):
     else:
         raise ValueError(f"unsupported Visual Studio version: {msvc_version}")
 
-    p = subprocess.check_output(
-        [
-            str(vswhere),
-            # Visual Studio 2019.
-            "-version",
-            version,
-            "-property",
-            "installationPath",
-            "-products",
-            "*",
-        ]
-    )
+    cmdline = [
+        str(vswhere),
+        # Visual Studio 2019.
+        "-version",
+        version,
+        "-property",
+        "installationPath",
+        "-prerelease",
+        "-products",
+        "*",
+    ]
+    p = subprocess.check_output(cmdline)
+    if not p:
+        cmd = " ".join('"%s"' % i for i in cmdline)
+        print("could not find visual studio (using '%s')" % cmd)
+        sys.exit(1)
 
-    # Strictly speaking the output may not be UTF-8.
-    p = pathlib.Path(p.strip().decode("utf-8"))
+    p = p.strip().decode("utf-8")
+    if "\n" in p:  # more than one line
+        print("found multiple matching instances of visual studio %s:" % msvc_version)
+        print(p)
+        sys.exit(1)
 
-    p = p / path
+    # Strictly speaking the output may not be UTF-8.
```

Comment:
Should this comment remain up where we decode the output?

Reply:
Yep, fixed in latest commit along with some additional comments.